internal/cache/file/downloader/job.go

```diff
@@ -133,13 +157,28 @@ func (job *Job) init() {
 // Acquires and releases LOCK(job.mu)
 func (job *Job) Cancel() {
 	job.mu.Lock()
-	defer job.mu.Unlock()
-	if job.status.Name == DOWNLOADING || job.status.Name == NOT_STARTED {
-		job.cancelFunc()
```

Comment:
What happens if we call the cancelFunc() twice?

Reply:
Nothing. https://pkg.go.dev/context#CancelFunc